YANG/ccamp/l1csm/ietf-l1csm.tree

```diff
@@ -3,10 +3,14 @@ module: ietf-l1csm
      +--rw access
      |  +--rw unis
      |     +--rw uni* [id]
-     |        +--rw id                   string
-     |        +--rw protocol?            identityref
-     |        +--rw coding?              identityref
-     |        +--rw optical-interface?   identityref
+     |        +--rw id                         string
+     |        +--rw (uni-access-type)?
+     |           +--:(mef)
+     |           |  +--rw protocol             identityref
+     |           |  +--rw coding               identityref
+     |           |  +--rw optical-interface    identityref
+     |           +--:(itu)
```

Comment:
Yes ITU is different with MEF, but has the same as IETF UNI. Shall we put the branch name as 'itu or ietf'? or something else to cover broader scope?